changelogs/unreleased/3952-syntax-highlight-elif-vim.yml

```diff
@@ -0,0 +1,5 @@
+description: Add highlighting for the elif keyword for vim
+change-type: minor
+destination-branches: [master, iso5]
+sections:
+    feature: "{{description}}"
```

Comment:
I wouldn't make this a feature. This is such a small change on such a little used thing that I really don't think this needs mention in the changelog.